harness/determined/keras/_tf_keras_trial.py

```diff
@@ -747,6 +747,9 @@ def _launch_evaluate(self) -> Any:
         use_model_metrics = version.parse(tf.__version__) < version.parse("2.2.0")
         evaluate_kwargs = {} if use_model_metrics else {"return_dict": True}
 
+        if self.context.env.test_mode:
```

Comment:
throughout: All these TrialController objects have a `self.env` that you should use directly.